libretroshare/src/serialiser/rsserializer.h

```diff
@@ -261,6 +255,12 @@ struct RsGenericSerializer : RsSerialType
 	static const SerializationFlags SERIALIZATION_FLAG_SIGNATURE;      // 0x0002
 	static const SerializationFlags SERIALIZATION_FLAG_SKIP_HEADER;    // 0x0004
 
+	/** Used for JSON deserialization in JSON API, it causes the deserialization
+	 * to continue even if some field is missing (or incorrect), this way the
+	 * API is more user friendly as some methods need just part of the structs
+	 * they take as parameters. */
+	static const SerializationFlags SERIALIZATION_FLAG_YIELDING;       // 0x0008
```

Comment:
This should obviously be restricted to JSon only. Using this in the binary serialization is very risky. Can you somewhere check this for consistency?